.github/workflows/ci.yml

```diff
@@ -14,32 +14,32 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     env:
-      min-python-version: '3.10'
+      min-python-version: "3.10"
 
     steps:
-    - name: Check out repository
-      uses: actions/checkout@v3
+      - name: Check out repository
```

Comment:
Is that spacing right - it's different to before?

Reply:
It’s how yaml is supposed to be formatted. Not sure why it wasn’t before